src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -308,7 +308,7 @@ private function renderCell(array $row, $column, $cellFormat)
         if ($cell instanceof TableSeparator) {
             $this->output->write(sprintf($this->style->getBorderFormat(), str_repeat($this->style->getHorizontalBorderChar(), $width)));
         } else {
-            $width += Helper::strlen($cell) - Helper::strlenWithoutDecoration($this->output->getFormatter(), $cell);
+            $width += Helper::strlen($cell) - Helper::strlenWithoutDecoration($this->output->getFormatter(), (string) $cell);
```

Comment:
IMO it would be better to cast to string in Helper::strlenWithoutDecoration and OutputFormatter::format